### PR TITLE
added week12-2 materials

### PR DESCRIPTION
**Pull request by bwolfson2**
Requesting a pull request for week 12 materials. I have added a en/docs/week12/12-2 folder as wells as the images/week12/12-2 folder

### CODE REVIEW

File: docs/en/week12/12-2.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
+---
+lang-ref: ch.13-2
```
Comment (Atcold) [inline docs/en/week12/12-2.md:2]: Is this week 13 or 12???

Comment (bwolfson2) [reply docs/en/week12/12-2.md:2]: I'm confused I thought I was responsible for 12-2 only. The summary is in the scribe notes. Apologies for the wrong lang-ref.

**Comment by Atcold** [comment]
The summary goes in the file `12.md`. Please, check out previous submissions.

**Comment by Atcold** [comment]
Screenshot 2020-05-04 at 01 20 10

I'm merging for now. Please, add the summary to `12.md`.

**Comment by Atcold** [comment]
Links to images are broken.

Screenshot 2020-05-04 at 01 23 14

**Comment by Atcold** [comment]
Screenshot 2020-05-04 at 01 29 06

Fix this entry (make it green) and add it to `12.md`.

**Pull request merged**